src/coreComponents/mesh/generators/PAMELAMeshGenerator.hpp

```diff
@@ -147,7 +147,29 @@ class PAMELAMeshGenerator : public MeshGeneratorBase
     static string retrieveSurfaceOrRegionName( string const & pamelaLabel )
     {
       string_array const splitLabel = stringutilities::tokenize( pamelaLabel, m_separator );
-      return splitLabel[splitLabel.size() -2 ];
+
+      // The PAMELA label looks like: PART00002_POLYGON_POLYGON_GROUP_Ovbd1_Ovbd2_14
+      // But we only want to keep Ovbd1_Ovbd2
+      // So we find the word GROUP, and then we keep what is after, except the last piece
+
+      using std::begin;
+      using std::end;
```

Comment:
I am not sure about the usage of the `using` keyword, can't we just put `std` in front all the `begin` and `end ?`

Reply:
Yes, I removed the `using` keyword in the last commit and just used `std::begin` and `std::end`